ipmininet/cli.py

```diff
@@ -81,6 +81,25 @@ def do_ping6pair(self, _line):
         """Ping (IPv6-only) between first two hosts, useful for testing."""
         self.mn.ping6Pair()
 
+    def do_linkFailure(self,line):
+        """Down the link between 2 nodes"""
+        nodes = line.split(' ')
+        if(len(nodes) != 2):
+            lg.output("Error: specify 2 nodes")
+        else:
+            self.mn.runFailurePlan([(nodes[0],nodes[1])])
```

Comment:
You defined an API enabling to fail multiple links, you should enable to interface directly with here; e.g.
linkFailure u v , w x
to fail two links.

This would require you to parse the line:
- first looking for link separators
- then for the link descriptions

This is particularly important as in this case this command is strictly equivalent to the already existing:
link u v down